pkg/controller/networkpolicy/crd_utils.go

```diff
@@ -35,6 +36,25 @@ var (
 	}
 )
 
+// CompareNetworkPolicyStatus compares two NetworkPolicyStatus objects, ignoring
+// LastTransitionTime and message equality in the status Conditions.
+func CompareNetworkPolicyStatus(oldStatus, newStatus v1alpha1.NetworkPolicyStatus) bool {
```

Comment:
Is it right to ignore LastTransitionTime and message? I think it will cause Status update not triggered when the fields don't match.

If it's right to ignore LastTransitionTime and message, you could refer to https://github.com/kubernetes/kubernetes/blob/8b6dd065d7f9b688fe565a2398065a23721a259b/pkg/controller/util/endpoint/controller_utils.go#L353-L355

Reply:
Decision is to ignore LastTransitionTime but keep message in the comparison.